core/templates/domain/exploration/SolutionObjectFactory.ts

```diff
@@ -130,6 +129,25 @@ export class Solution {
       '". ' + explanation + '.');
   }
 
+  // Function to get the answer of Drag and Drop Sort Input interaction.
+  getDragAndDropAnswer(
+      interactionId: 'DragAndDropSortInput',
+      customizationArgs: InteractionCustomizationArgs): string {
+    var correctAnswer = null;
+    correctAnswer = [];
+    customizationArgs = (
+      <DragAndDropSortInputCustomizationArgs> customizationArgs
+    ).choices.value.map(
+      choice => choice.html);
+    let transformedArrayone = [];
+    var transformedArraytwo = transformedArrayone.concat(customizationArgs);
+    for (let elem of transformedArraytwo) {
+      let transformedArray = [];
+      transformedArray.push(elem);
+      correctAnswer.push(transformedArray);
```

Comment:
Seems like `correctAnswer` will be a list of single-item lists? Is this expected? What happens when there are  "Allow multiple sort items in the same position" is set in the customisation args and there are overlapping items in the solution? Has this been tested manually?